submissions/israelOladosu/DomManipulation/Script.js

```diff
@@ -0,0 +1,96 @@
+let submitForm = document.getElementById("validateForm"),
+  email = document.getElementById("email"),
+  password = document.getElementById("password"),
+  confirmPassword = document.getElementById("pswd"),
+  firstName = document.getElementById("firstName"),
+  lastName = document.getElementById("lastName"),
+  otherName = document.getElementById("otherName"),
+  phoneNumber = document.getElementById("phone");
+
+var regEx = {
+  name: /[^A-Za-z]/,
+  password: /^(?=.*[A-Z])(?=.*[a-z])(?=.*[@$!%*?&_<>':;])(?=.*[\d])[a-zA-Z0-9@$!%*?&_<>':;]{6,}$/,
+  email: /^([a-z\d\.-]+)@([a-z\d-]+)\.([a-z]{2,8})$/i,
+  phoneNumber: /^[\+0-9]{11,}$/,
+};
+
+function required(field, e) {
+  if (field.id == "pswd" && field.value == "") {
+    field.nextElementSibling.innerHTML = `Please confirm your password`;
+  } else if (field.value == "") {
+    field.nextElementSibling.innerHTML = `Please enter your ${field.previousElementSibling.innerText}`;
+    e.preventDefault();
+    return false;
+  } else if (
```

Comment:
I will suggest you handle this regex in a different function

Reply:
Okay... I will